src/index.ts

```diff
@@ -0,0 +1,134 @@
+import escape from 'xml-escape';
+import * as Cypress from 'cypress';
+import BeforeRunDetails = Cypress.BeforeRunDetails;
+import PluginConfigOptions = Cypress.PluginConfigOptions;
+import PluginEvents = Cypress.PluginEvents;
+import Spec = Cypress.Spec;
+import { ConfigOption, JUnitTestSuite } from './type';
+import Reporter, { TestCase, TestSuite } from './reporter';
+
+let reporter: Reporter;
+
+const onBeforeRun = (details: BeforeRunDetails) => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  reporter.junitSuite.name = `Cypress Test - ${(details.config as any)?.projectName} ${details.browser?.displayName}`;
```

Comment:
Hm. `projectName` is not officially documented. I'm not necessarily against using this, but then we should mention that in our docs. No one will set this value otherwise. For me, by default, this value is populated as `v1`. I'm sort of curious if this `projectName` is also dynamically resolved, if the user uses a Cypress Cloud projectID and has a name defined there for said project.

Reply:
From my test and Cypress source code, `projectName` is the basename of the current folder. So I choose this one to indicate its workspace.

https://github.com/cypress-io/cypress/blob/develop/packages/data-context/src/data/ProjectConfigManager.ts#L509

Reply:
😂 That explains why I was seeing `v1`, having tested it with our [Cypress example repo](https://github.com/saucelabs/saucectl-cypress-example/tree/main/v1).

Reply:
Exactly